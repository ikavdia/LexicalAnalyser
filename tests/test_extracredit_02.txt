ABCDEF1324325x16as
ABCDEFGx016