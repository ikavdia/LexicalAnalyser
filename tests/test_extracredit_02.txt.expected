{ABCDEF1324325x16as , ID , 1}
{ABCDEFGx016 , ID , 2}
{ , END_OF_FILE , 2}